Orange/widgets/visualize/utils/customizableplot.py

```diff
@@ -273,6 +231,8 @@ def update_axis(axis, **settings):
             }
         }
 
+        self.initial_settings = {}
```

Comment:
You can use

`initial_settings: Dict[str, Dict[str, SettingsType]] = NotImplemented`

if you want to keep the old functionality, but I'm fine with any solution.

Reply:
Thanks!